tests/MongoDB.Driver.Core.Tests/Core/Operations/EndTransactionOperationTests.cs

```diff
@@ -149,8 +184,9 @@ public class AbortTransactionOperationTests
         public void constructor_should_initialize_instance()
         {
             var writeConcern = new WriteConcern();
+            var recoveryToken = BsonDocument.Parse("{ generalOrder: 1 }");
 
-            var result = new AbortTransactionOperation(writeConcern);
+            var result = new AbortTransactionOperation(recoveryToken, writeConcern);
```

Comment:
Assert below that recoverToken was saved:

    result._recoveryToken().Should().BeSameAs(recoveryToken);

Reply:
Fixed